calico-policies/private-network-isolation/calico-v3/au-syd/allow-private-services-pods.yaml

```diff
@@ -12,18 +13,39 @@ spec:
     destination:
       nets:
       # IBM Cloud Container Registry: Global registry
-      - 166.9.20.4/32
-      - 166.9.22.3/32
-      - 166.9.24.2/32
+      - 166.9.20.31/32
+      - 166.9.22.22/32
+      - 166.9.24.16/32
       # IBM Cloud Container Registry: AP South nets for your own registry
-      - 166.9.52.2/32
-      - 166.9.54.2/32
-      - 166.9.56.3/32
+      - 166.9.52.20/32
+      - 166.9.54.19/32
+      - 166.9.56.13/32
       ports:
       - 443
       - 4443
     protocol: TCP
     source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM Cloud Monitoring with Sysdig: AU Syd nets
+      - 166.9.52.27/32
+      - 166.9.54.27/32
+      - 166.9.56.32/32
+      ports:
+      - 6443
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM Log Analysis with LogDNA: AU Syd nets
+      - 166.9.56.20/32
+      - 166.9.52.5/32
+      ports:
+      - 443
```

Comment:
Still need confirmation that `443` is he correct port for logdna private endpoint ips.